### PR TITLE
Add artifacts to test runs

### PR DESCRIPTION
**Pull request by seancribbs**
To facilitate the below two use-cases, we should add the ability to upload "artifacts" to completed test runs:
1. When a test fails, we want to upload the log files of each node involved.
2. When submitting a performance test, it's important to be able to upload or link to the graph output of the test.

/cc @matthewvon @evanmcc @joedevivo @cmeiklejohn

### CODE REVIEW

**Comment by seancribbs** [comment]
Requesting review from @cmeiklejohn. I'm aware that things are broken in a few places (e.g. load a deep test_result/artifact link but then go back to the scorecard), but we need artifacts and the UI restructuring badly.

File: Gemfile [hunk 1]
```diff
@@ -17,5 +17,4 @@ group :assets do
   gem 'rake-pipeline-web-filters'
   gem 'uglifier'
   gem 'tilt'
-  # gem 'therubyracer', :platforms => :ruby
```
Comment (cmeiklejohn) [inline Gemfile:20]: Why is this removed?

File: assets/javascripts/app/app.js [hunk 2]
```diff
@@ -1,12 +1,13 @@
-GiddyUp = Ember.Application.create();
+/* global GiddyUp:true */
+GiddyUp = Ember.Application.create({
+  LOG_TRANSITIONS: true
```
Comment (cmeiklejohn) [inline assets/javascripts/app/app.js:3]: Do we want to push with this in there or take it out once the review is done?

Comment (seancribbs) [inline assets/javascripts/app/app.js:3]: Let's take it out after review.

File: Gemfile (hunk 1 above)
Comment (seancribbs) [inline Gemfile:20]: It didn't work on the bos1 machines, and they have node.js anyway so you don't have to use rubyracer.

Comment (cmeiklejohn) [inline Gemfile:20]: Cool, thanks.

**Comment by cmeiklejohn** [comment]
:+1: looks good to me, asset display seems to work perfectly as far as I can tell.
